Dockerfile

```diff
@@ -16,9 +16,9 @@ COPY setup.py .
 COPY main.py .
 
 # Copy relevant files to run bot
-COPY config.yml .
-COPY old_coins.json .
-COPY auth/auth.yml ./auth/
+COPY config.example.yml config.yml
```

Comment:
I'm not too familiar with Docker but can you easily edit files afterwards once you created the Docker?

The problem I'm seeing here, if you can't edit easily inside the docker.,is that you would need to edit the .example files beforehand and then create the Docker Container.
This would then cause problems when you try to update the git repo (git pull) by causing merge conflicts, since the .example files are tracked by git.

I would prefer to stick with the old files: config.yml, auth.yml and old_coins.json, that way you're not forced to edit the tracked .example files.

Reply:
The other problem now is that image won't be built cause the git repo doesn't contain the necessary config files.

Config files should be provided from the outside, they can be mounted from any path of the host. Or someone can use default settings and provide secrets through the ENV. Provided from the outside config files doesn't affect git repo in any way. You can take a look at the Docker storage concept [here](https://docs.docker.com/storage/)

Moreover, we can publish the app image in the GitHub repo image registry. So it will be no necessity to build an image for use, you could just pull it from the official repo registry.

Reply:
If you provide you config before building, you have to rebuild you image after every config change.
Or you can edit files in the container, it's not a big deal.  But when you kill the container and start it again all your edits would be lost

Reply:
I read through the link you proved a bit and like the idea of using volumes/mounting to separate the config from the container.
We should also consider #130 to make it easier.
I would like to rework the whole Docker structure and make it properly useable/configurable.

For that I need to take a closer look at Docker first but the problem is that I'm currently busy with life for the next two weeks or so.


Regarding this PR. I would like to keep the current Dockerfile the same, as to not suddenly change the behavior (what config.yml/auth.yml file is needed) and in a new PR we can rework the whole Docker integration to utilize mounting or volumes (after I familiarized myself more). 

Therefore this PR will just merge the environment variable changes.
- [ ] Revert path changes to Dockerfile

Whats your opinion on this?

Reply:
If the config files were moved to a single directory it would be not so much to reorder in the Dockerfile. But moving them is up to the app.
You are free to revert Dockerfile. It's just the setup I personally use at the moment, as long as the config files is not reordered.

Reply:
I don't fully understand what you mean by "moving them up to the app"

As I understand it we would use the (then existing) config directory (`<root>/config`) to have the config/auth.yml file in one directory and then mount that directory into the docker container.
That way we can change the config file and just restart the container/bot and it will apply the changes without use needing to rebuild the whole container.

In that case we would still call the files option.yml and auth.yml and keep the .example files as they are, so source control wouldn't mark them as changed.
The options.yml and auth,yml are in .gitignore so you can configure them freely.

I will take a closer look at this once i have more time on my Hand.

Reply:
I meant we need change the app code to move config files.